### PR TITLE
Removed wget in favor of curl

### PR DESCRIPTION
**Pull request by tdensmore**
@jdewinne this is the PR for removing `wget` and adding`curl`

### CODE REVIEW

**Review by jdewinne — APPROVED** [review]
lgtm

**Comment by tdensmore** [comment]
@AmberAlston I managed to get a LGTM ... do I need to merge this, or should I leave that to you?

**Comment by AmberAlston** [comment]
Thanks @tdensmore and @jdewinne ! I'll merge

**Pull request merged**
